src/modules/appeals-service/errors.ts

```diff
@@ -1,8 +1,29 @@
+import { NOT_FOUND, UNAUTHORIZED, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+
+interface StatusCode {
+    status: number;
+}
 // tslint:disable: max-classes-per-file
-export class AppealNotFoundError extends Error {}
 
-export class AppealServiceError extends Error {}
+export class AppealServiceError extends Error implements StatusCode {
+    public readonly status: number;
```

Comment:
It does not make much sense to have status codes on service errors. It is down to app to interpret errors and react accordingly. Service should hide HTTP concerns from API consumers.